subsys/net/lib/lwm2m_client_utils/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_DEVICE_OBJ_SUPPORT lwm2m/
 zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_SECURITY_OBJ_SUPPORT lwm2m/lwm2m_security.c)
 zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT
 			     lwm2m/ecid_signal_meas_info.c)
-zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_NEIGHBOUR_CELL_LISTENER
-			     lwm2m/neighbour_cell_info.c)
+zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS
+			     lwm2m/lwm2m_lte_notification.c)
```

Comment:
To avoid the addition of `lwm2m_lte_notification.c` to the build when `CONFIG_LWM2M_CLIENT_UTILS_NWK_REG_NOTIFICATION` and `CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT` are not used, please create a KConfig flag which is enabled if either of the flags is enabled and use that instead here.

Reply:
OK I will do generate Kconfig flag which is enabled if CONFIG_LWM2M_CLIENT_UTILS_NWK_REG_NOTIFICATION or CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT is activated.